Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -1201,3 +1203,10 @@ class SearchBarTextField: UITextField, PrivateModeUI {
         return bounds.insetBy(dx: SearchBarTextField.leftInset, dy: 0)
     }
 }
+
+private extension URL {
+    init?(nullableString: String?) {
+        guard let nonNullString = nullableString else { return nil }
+        self.init(string: nonNullString)
+    }
+}
```

Comment:
I recommend 

1. putting this code into a file called `URL+Nullable.swift` (this is a UIKit dev convention)
2. adding a one liner for documentation and
3. making it public. 

I'm sure we can use this again.